spec/helpers/mailer_helper_spec.rb

```diff
@@ -11,6 +11,20 @@
     end
   end
 
+  describe "#creation_title" do
```

Comment:
If you could also check when creation is a series, everything will be covered and this should be good to go!